gradle.properties

```diff
@@ -1,8 +1,8 @@
-org.gradle.jvmargs=-Xmx1G
+org.gradle.jvmargs=-Xmx3G
 org.gradle.daemon=false
-minecraft_version=1.18.1
+minecraft_version=1.18.2
 archives_base_name=ferritecore
-mod_version=4.1.2
+mod_version=4.1.3
```

Comment:
I'll merge this anyway because it's easier for me to quickly fix it myself, but please:
1. Please don't update the version number in PRs. I'll probably make this 4.2.0 as it's moving across a non-compatible MC version change, even though it's not a full version.
2. The memory limit of 1 GB was intentional, I try to keep the memory used by my dev environment down if I can. Did you run into issues with that limit or why did you change it?
3. You should probably have updated the MC version requirement in `fabric.mod.json` and `mods.toml` as well.

Reply:
> Did you run into issues with that limit or why did you change it?

Yes, it ended up running out of memory a few times when I tried building. Only increasing the limit would allow me to build.

As for everything else, my apologies.

Reply:
For the first part, I noticed that myself when I actually did the build: da1247a9ca9d38933ef6607d62b646640d965aa7. Some update in the build setup (I suspect loom) causes all subprojects to build at once, which I don't actually care about (build time is fine with 1 project at a time); and of course building 2-3 things at once uses more memory than building only one.

Thanks for the PR in general, forgot that yesterday!